content/docs/concepts/architecture.md

```diff
@@ -0,0 +1,320 @@
+to 'refresh' the {{<color "#6A00FF" >}}vWire{{< /color >}}.  If a {{<color "#6A00FF" >}}vWire{{< /color >}} exceeds its expire time without being refreshed, NSM cleans up the {{<color "#6A00FF" >}}vWire{{< /color >}}.
+
+A {{<color "#0050EF" >}}Client{{< /color >}} may use a 'MonitorConnection' streaming GRPC call to NSM to get updates on the status of a {{<color "#6A00FF" >}}vWire{{< /color >}} it has to a Network Service.
+
+## Registries
+
+As with any other Mesh, Network Service Mesh has Network Service Registries (NSR) in which Network Services and Network Service {{<color "#008A00" >}}Endpoint{{< /color >}}s are registered
+
+### Network Service Endpoint
+
+A {{<color "#008A00" >}}Network Service Endpoint{{< /color >}} ({{<color "#008A00" >}}NSE{{< /color >}} or {{<color "#008A00" >}}Endpoint{{< /color >}}) provides one or more Network Services.  It registers with the registry a list
+of Network Services (by name) that it provides, and the 'destination labels' it is advertising for each Network Service.
+
+
+### Network Service
+
+A Network Service is a identified by name, and carries a payload type (either IP or Ethernet).
+
+```yaml
+---
+apiVersion: networkservicemesh.io/v1
+kind: NetworkService
+metadata:
+  name: simple-network-service
+spec:
+  payload: IP
+```
+
+by default if not specified, the payload is presumed to be IP.  Network Services are registered with the Network Service Registry.
+
+Optionally, a Network Service may specify a list of 'matches'.  These matches allow matching a the 'source labels' a {{<color "#0050EF" >}}Client{{< /color >}} sends
+with its Request to 'destination labels' advertised by the {{<color "#008A00" >}}Endpoint{{< /color >}} when it registers as providing the Network Service.
+
+For example:
+```yaml
+---
+apiVersion: networkservicemesh.io/v1
+kind: NetworkService
+metadata:
+  name: service-mesh@finance.example.com
+spec:
+  payload: IP
+  matches:
+    - source_selector:
+        service: envoy-proxy
+      routes:
+        - destination_selector:
+            service: vl3
+    - source_selector:
+      routes:
+      - destination_selector:
+          service: envoy-proxy
+```
+
+If a {{<color "#0050EF" >}}Client{{< /color >}} provided no 'source labels' with its Request for the 'service-mesh' Network Service, it would not match the 'service: envoy-proxy'
+for the first match, and so would fall through to the final 'catch all' match with no source_selector, and be matched to {{<color "#008A00" >}}Endpoint{{< /color >}}
+that advertised a Network Service named 'service-mesh' with 'destination label' 'service: envoy-proxy'.
+
+If a {{<color "#0050EF" >}}Client{{< /color >}} provided a 'source label' of 'service: envoy-proxy' it would match the first match and be matched to an {{<color "#008A00" >}}Endpoint{{< /color >}}
```

Comment:
Add comma.  "If a Client provided a 'source label' of 'service: envoy-proxy', ...